scripts/zulip-notification/index.js

```diff
@@ -0,0 +1,65 @@
+// index.js
+
```

Comment:
Should we publish the script on npm so that other projects can reuse the module?

Reply:
@mannycolon Lets make that a separate collaborative project for the developers.